.github/workflows/release-please.yml

```diff
@@ -0,0 +1,26 @@
+name: release-please
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
This package has been deprecated, if you can please update it to `googleapis`

Reply:
https://github.com/googleapis/release-please-action?tab=readme-ov-file#basic-configuration

The readme includes a comment about setting up a PAT token as the secret